src/services/activities/activities.hooks.js

```diff
@@ -1,14 +1,15 @@
 const { authenticate } = require('@feathersjs/authentication').hooks;
+const permission = require('../../hooks/permission');
 
 module.exports = {
   before: {
     all: [authenticate('jwt')],
     find: [],
     get: [],
-    create: [],
-    update: [],
-    patch: [],
-    remove: [],
+    create: [permission({ roles: ['admin'] })],
```

Comment:
Should managers also be able to create activities? (E.g. Regional group having different activities)